### ISSUE TITLE
measurement page: add support for showing the network name

### ISSUE THREAD

**Issue by hellais** [ooni/explorer#45]
Currently we only show the ASN, but we don't show the network name as this is not being exposed via the OONI API or submitted as part of the measurement.

This ticket is about extending OONI Explorer to also resolve a ASN into a proper network name.

**Comment by darkk** [comment]
FYI, there are other sorts of inconsistencies as well. E.g. `originas` zone from `routeviews` can produce following records:

```
  asn  |    origin     |                      asname
-------+---------------+---------------------------------------------------
 17916 | 20.134.0.0/20 | CSC-IGN-AUNZ-AP Computer Sciences Corporation, AU
  7474 | 20.134.0.0/20 | OPTUSCOM-AS01-AU SingTel Optus Pty Ltd, AU
```

**Comment by hellais** [comment]
I don't think we will manage to do this in the stable as it requires a fair amount of backend and pipeline work as well.

**Comment by sarathms** [comment]
In that case, I think we should change the layout of the relevant section and make it a 3 column layout on desktop like below. Significantly changes the first impression of the page. @holantonela @hellais 

![image](https://user-images.githubusercontent.com/700829/59203481-a572be00-8b6c-11e9-9083-2ff489988aea.png)

It is already one-column on mobile.
![image](https://user-images.githubusercontent.com/700829/59203619-f2569480-8b6c-11e9-80b3-cbdf88595d4f.png)

**Comment by holantonela** [comment]
I suggested this before, but @hellais wants to have room for the long `asname`. Could we stick to the three columns until we have the `asname` available @hellais?

**Comment by hellais** [comment]
This may be relevant to the work @FedericoCeratto is doing with the ASN database.

**Comment by hellais** [comment]
We might want to generate this using the asn-datatbase generator code here: https://github.com/ooni/asn-db-generator or potentially put it in a timer inside of the pipeline.

**Comment by hellais** [comment]
We now have this field inside of the probe generated measurements directly. Perhaps we can start displaying it for measurements that include it and leave it blank in case we don't spot it.

Eventually the pipeline should backfill them.

**Comment by hellais** [comment]
We can just add it to the header and leave the AS in parenthesis like so:
![Screenshot 2022-08-04 at 16 12 19](https://user-images.githubusercontent.com/424620/182868814-5f4456f0-696c-4c77-a223-92dd369e4715.png)

**Comment by hellais** [comment]
This is fixed and will be addressed better by the new pipeline.
